code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -1,184 +1,153 @@
 /mob/living/simple_animal/hostile/zombie
 	name = "zombie"
-	desc = "A horrifying, shambling carcass of what once resembled a human being."
+	desc = "When Observe is full, the dead shall walk the station."
 	icon = 'icons/mob/human.dmi'
 	icon_state = "zombie_s"
 	icon_living = "zombie_s"
 	icon_dead = "zombie_dead"
 	turns_per_move = 5
-	emote_see = list("groans", "roars")
+	speak_emote = list("groans")
+	emote_see = list("groans")
 	a_intent = "harm"
 	maxHealth = 120
 	health = 120
 	speed = 2
-	move_to_delay = 5
 	harm_intent_damage = 8
-	melee_damage_lower = 9
+	melee_damage_lower = 20
 	melee_damage_upper = 20
 	attacktext = "claws"
-	attack_sound = list('sound/weapons/claw_strike1.ogg','sound/weapons/claw_strike2.ogg','sound/weapons/claw_strike3.ogg')
+	attack_sound = 'sound/hallucinations/growl1.ogg'
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
-	ignored_damage_types = list(BRUTE = 0, BURN = 0, TOX = 1, CLONE = 1, STAMINA = 1, OXY = 1)
 	minbodytemp = 0
 	maxbodytemp = 350
 	unsuitable_atmos_damage = 10
-	environment_smash = 1 //Can smash tables and lockers, etc.
-	can_force_doors = 1 //Can force doors open like a champ
+	environment_smash = 1
 	robust_searching = 1
-	stat_attack = 1 //Can attack unconscious players
-	gold_core_spawnable = 0 //No.
+	stat_attack = 2
+	gold_core_spawnable = 1
```

Comment:
yeah great job now old /tg/ zombies will be able to ruin rounds with gold cores once more